### PR TITLE
Feature/e UI 3565 judicial user header

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3565

### Change description ###

Tabs for the WA

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/app/components/hmcts-global-header/hmcts-global-header.component.ts [hunk 1]
```diff
@@ -27,9 +27,19 @@ export class HmctsGlobalHeaderComponent {
   public showItems: boolean;
   public userValue = true;
   public tab;
+  public leftItems: NavItemsModel[];
+  public rightItems: NavItemsModel[];
 
   constructor(
     public nocStore: Store<fromNocStore.State>) { }
+  ngOnChanges(changes: SimpleChanges): void {
```
Comment (kuda1992) [inline src/app/components/hmcts-global-header/hmcts-global-header.component.ts:35]: no line break???

**Review by leepavlouhmcts — APPROVED** [review]
looks good

File: src/app/components/hmcts-global-header/hmcts-global-header.component.ts [hunk 2]
```diff
@@ -27,9 +27,19 @@ export class HmctsGlobalHeaderComponent {
...
+    this.splitNavItems();
+  }
+
+  public splitNavItems() {
+    this.rightItems = this.items.filter(item => item.align && item.align === 'right');
```
Comment (kuda1992) [inline src/app/components/hmcts-global-header/hmcts-global-header.component.ts:40]: `splitNavItems()` runs when any of the `@Input()` properties change. Can we make  `splitNavItems()` run only when the interested property changes are made?

`if(changes['items']) { // rest of the code}
`

**Review by kuda1992 — COMMENTED** [review]
Almost there just few changes

File: src/app/components/hmcts-global-header/hmcts-global-header.component.ts (hunk 2 above)
Comment (udaydenduluri33) [reply src/app/components/hmcts-global-header/hmcts-global-header.component.ts:40]: Good point. Fixed it now

File: src/app/components/hmcts-global-header/hmcts-global-header.component.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/app/components/hmcts-global-header/hmcts-global-header.component.ts:35]: Done

**Review by kuda1992 — APPROVED** [review]
looks good now

**Pull request merged**